app/src/main/res/values/dimens.xml

```diff
@@ -46,8 +46,8 @@
 
     <dimen name="fab_margin">16dp</dimen>
 
-    <dimen name="remote_cursor_size">45dp</dimen>
-    <dimen name="remote_cursor_size2x">90dp</dimen>
+    <dimen name="remote_cursor_size">24dp</dimen>
+    <dimen name="remote_cursor_size2x">48dp</dimen>
```

Comment:
nit: this actually seems unused (I wonder why we needed it?)

Reply:
Not sure! I just figured I would update it for correctness